include/boost/variant/get.hpp

```diff
@@ -22,9 +22,13 @@
 #include <boost/utility/addressof.hpp>
 #include <boost/variant/variant_fwd.hpp>
 #include <boost/variant/detail/element_index.hpp>
+#include <boost/variant/detail/move.hpp>
 
 #include <boost/type_traits/add_reference.hpp>
 #include <boost/type_traits/add_pointer.hpp>
+#include <boost/type_traits/is_lvalue_reference.hpp>
+
+#include <boost/mpl/not.hpp>
```

Comment:
Use this header if you are doing some heavy metaprogramming and need lazy evaluation. Otherwise '!' is better.